locale/es_ES.po

```diff
@@ -695,7 +694,7 @@ msgstr "Código DTCS"
 #: ../chirpui/memedit.py:223 ../chirpui/memedit.py:878
 #: ../chirpui/memedit.py:933 ../chirpui/memedit.py:1060
 msgid "DTCS Pol"
-msgstr "DTCS Pol"
+msgstr "Política DTCS"
```

Comment:
Hola, mi espanol no es tan bueno, pero en este linea, `Pol` debe estar `polaridad`, no?
https://chirp.danplanet.com/projects/chirp/wiki/MemoryEditorColumns#DTCS-Pol

Si ves pagina 50 de https://www.manualslib.es/manual/114361/Icom-Ic-V80.html, busca `polaridad DTCS`

--

In case you speak English :P, I think that `polaridad` is better. Everything else looks good to me.

Reply:
yes, fixed, thanks :P